utils/merge-rulesets.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/sh
 
-# Merge all the .xml rulesets into a single "default.rulesets" file -- this
-# prevents inodes from wasting disk space, but more importantly, works around
-# the fact that zip does not perform well on a pile of small files.
+# Merge all the .xml rulesets into a single "default.rulesets" file -- this prevents inodes from wasting disk space, but more importantly,
+# works around the fact that zip does not perform well on a pile of small files.
 
 python2.7 ./utils/merge-rulesets.py
 
 RULESETS=chrome/content/rules/default.rulesets
+
+# We make default.rulesets at build time, but it shouldn't have a variable timestamp.
+touch -r src/install.rdf src/$RULESETS
```

Comment:
I don't see how this is a substantive improvement.  Care to explain?

Reply:
I moved this out of the Python script into the shell script.